x-pack/plugins/apm/public/components/shared/managed_table/index.tsx

```diff
@@ -21,6 +21,7 @@ export interface ITableColumn<T> {
   align?: string;
   width?: string;
   sortable?: boolean;
+  truncateText?: boolean;
```

Comment:
is this being used somewhere?

Reply:
Yes, in https://github.com/elastic/kibana/blob/9d42d670e1676a622d7ee7d4d564c4d7dc3973e8/x-pack/plugins/apm/public/components/app/Settings/customize_ui/custom_link/custom_link_table.tsx#L41

We didn't have typed table columns before so it wasn't an error until now